tests/test_it_set.py

```diff
@@ -0,0 +1,38 @@
+#! pytest
+
+
+def test_insert(test_it_set_contract, accounts, web3):
+    contract = test_it_set_contract
+    tx_hash = contract.functions.testInsert(accounts[0]).transact({"from": accounts[0]})
+    tx_receipt = web3.eth.getTransactionReceipt(tx_hash)
+
```

Comment:
I guess this should test that accounts[0] is now inserted with `assert contract.functions.testContains(accounts[0]).call() is True`. Even though this is implicitly tested below.

Reply:
done